services/docker-compose.yml

```diff
@@ -358,8 +358,7 @@ services:
       - default
 
   postgres:
-    image: "postgres:10.11@sha256:2aef165ab4f30fbb109e88959271d8b57489790ea13a77d27\
-      c02d8adb8feb20f"
+    image: "postgres:10.11@sha256:2aef165ab4f30fbb109e88959271d8b57489790ea13a77d27c02d8adb8feb20f"
```

Comment:
just a thought but I wonder if we should not update that one once... current version is 14.x

Reply:
@sanderegg please add a maintenance case for it and we can follow up